docs/tools/vega-wallet/cli-wallet/latest/guides/isolate-keys.md

```diff
@@ -0,0 +1,250 @@
+ID:
+"VALIDATOR_ID"
+```
+
+#### For your original wallet:
+
+<Tabs groupId="operating-systems">
+<TabItem value="windows" label="Windows">
+
+```bash
+vega wallet info --wallet="MY_WALLET_NAME"
+```
+
+</TabItem>
+<TabItem value="mac" label="MacOS">
+
+```bash
+./vega wallet info --wallet="MY_WALLET_NAME"
+```
+
+</TabItem>
+<TabItem value="linux" label="Linux">
+
+```bash
+./vega wallet info --wallet="MY_WALLET_NAME"
+```
+
+</TabItem>
+</Tabs>
+
+#### Response 
+
+You'll receive the following information as a response. 
+
+```
+Type:
+HD wallet
+"1" or "2" 
+ID:
+"VALIDATOR_ID"
+```
+
+## Use isolated key in node wallet
+
+Validators who have isolated a key pair will then need to import that key to their node wallet. 
+
+You'll be prompted to enter passphrases for the node wallet and the blockchain wallet.
+  
+### 1. Import key to node wallet
+
+<Tabs groupId="operating-systems">
+<TabItem value="windows" label="Windows">
+
+```bash
+vega nodewallet import --force --chain=vega --wallet-path="ISOLATED_WALLET_PATH"
+```
+
+</TabItem>
+<TabItem value="mac" label="MacOS">
+
+```bash
+./vega nodewallet import --force --chain=vega --wallet-path="ISOLATED_WALLET_PATH"
+```
+
+</TabItem>
+<TabItem value="linux" label="Linux">
+
+```bash
+./vega nodewallet import --force --chain=vega --wallet-path="ISOLATED_WALLET_PATH"
+```
+
+</TabItem>
+</Tabs>
+
+#### Response
+
+You'll receive the following information as a response for a successful import. 
+
+```
+import successful:
+walletFilePath:
+"FILE_PATH"
+registryFilePath:
+"FILE_PATH"
+```
+
+### 2. Generate payload
+Then, generate the validator payload to be used on the Tendermint genesis file.
+
+You'll be prompted to enter the node wallet passphrase. 
+
+<Tabs groupId="operating-systems">
+<TabItem value="windows" label="Windows">
+
+```bash
+vega genesis new validator --country="XX" --info-url="VALIDATOR_URL" --name="NODE_NAME"
+```
+
+</TabItem>
+<TabItem value="mac" label="MacOS">
+
+```bash
+./vega genesis new validator --country="XX" 
+/ --info-url="VALIDATOR_URL" --name="NODE_NAME"
+```
+
+</TabItem>
+<TabItem value="linux" label="Linux">
+
+```bash
+./vega genesis new validator --country="XX" 
+/ --info-url="VALIDATOR_URL" --name="NODE_NAME"
+```
+
+</TabItem>
+</Tabs>
+
+### 3. Add details to genesis file
+
+Add the following information to the genesis file under `validators` key:
+```
+{
+  "address": "TENDERMINT_ADDRESS",
+  "pub_key": {
+    "type": "tendermint/PubKeyEd25519",
+    "value": "TENDERMINT_PUBLIC_KEY"
+  },
+  "power": "10",
+  "name": ""
+}
+```
+
+Add the following information into the genesis file under `app_state.validators` key:
+
+```
+{
+    "TENDERMINT_PUBLIC_KEY": {
+      "id": "VALIDATOR_ID",
+      "vega_pub_key": "ISOLATED_PUBLIC_KEY",
+      "ethereum_address": "MY_ETHEREUM_ADDRESS",
+      "tm_pub_key": "TENDERMINT_PUBLIC_KEY",
+      "info_url": "VALIDATOR_URL",
+      "country": "XX",
+      "name": "NODE_NAME",
+      "avatar_url": ""
+    }
+  }
+```
```

Comment:
Thought you might say that. I put them together as a temporary option, to get it out without having to create a whole new section with just that one guide, as it's a little bit out of context without more guides. I'm willing to break them up if you think it's too confusing or irrelevant. Though... it will also take a little longer to get this one out if so :)

Reply:
Alright, then it's fine.